.github/workflows/preview.yaml

```diff
@@ -39,26 +39,10 @@ jobs:
         id: eas-update
         run: |
           output=`eas update --branch=pr-${{ github.event.number }} --message '${{ github.event.pull_request.title }}' --non-interactive --json | jq -c`
-          echo $output
-          ios_id=`echo $output | jq '.[] | select(.platform == "ios") | .id' | tr -d '"'`
-          echo $ios_id
-          android_id=`echo $output | jq '.[] | select(.platform == "android") | .id' | tr -d '"'`
-          echo $android_id
-          echo "ios_url=exp://u.expo.dev/update/$ios_id" >> $GITHUB_OUTPUT
-          echo "android_url=exp://u.expo.dev/update/$android_id" >> $GITHUB_OUTPUT
-          cat $GITHUB_OUTPUT
-
-      - name: Set color
-        id: random-color-generator
-        run: echo "SELECTED_COLOR=green" >> $GITHUB_OUTPUT
-
-      - name: Get color
-        run: echo "The selected color is ${{ steps.random-color-generator.outputs.SELECTED_COLOR }}"
-
-      # - name: 💬 Comment preview
-      #   uses: expo/expo-github-action/preview-comment@v7
-      #   with:
-      #     channel: pr-${{ github.event.number }}
+          ios_id=`echo $output | jq -r '.[] | select(.platform == "ios") | .id'`
```

Comment:
nit: modern shell-isms would have this as

```sh
echo "ios_id=$( jq --raw-output '.[] | select(.platform == "ios" ) | .id ' <<<"${output}" )" >> "${GITHUB_OUTPUT}"
```

Reply:
sorry, did you just use "modern" and "shell" in the same sentence 😛

Reply:
TIL about `<<<`

Reply:
After putting shell into prod you learn tips and tricks to remove some of the sharp edges